internal/character/blade/talent.go

```diff
@@ -0,0 +1,66 @@
+package blade
+
+import (
+	"github.com/simimpact/srsim/pkg/engine/event"
+	"github.com/simimpact/srsim/pkg/engine/info"
+	"github.com/simimpact/srsim/pkg/key"
+	"github.com/simimpact/srsim/pkg/model"
+)
+
+const (
+	Talent       key.Attack   = "blade-talent"
+	IsAttack     key.Modifier = "blade_is_Attack"
+	GainedCharge key.Modifier = "blade_gained_charge"
+)
+
+func (c *char) Talent() {
+	e6AddMod := 0.0
+
+	// E6
+	if c.info.Eidolon >= 6 {
+		e6AddMod = 0.5
+	}
+
+	// Follow-up Attack
+	c.engine.InsertAbility(info.Insert{
+		Execute: func() {
+			c.engine.Attack(info.Attack{
+				Key:        Talent,
+				Source:     c.id,
+				Targets:    c.engine.Enemies(),
+				DamageType: model.DamageType_WIND,
+				AttackType: model.AttackType_INSERT,
+				BaseDamage: info.DamageMap{
+					model.DamageFormula_BY_ATK:    talentAtk[c.info.TalentLevelIndex()],
+					model.DamageFormula_BY_MAX_HP: (talentHP[c.info.TalentLevelIndex()] + e6AddMod),
+				},
+				StanceDamage: 30.0,
+				EnergyGain:   10.0,
+			})
+		},
```

Comment:
* 3 hit attack, 33% hit split ratio
* `AttackEnd` after the three hits
* Heal happens after the attack
* Charge count resets after the heal (its possible for blade to hit an edge case where the insert gets cancelled before it happens. In this instance charge count does not get reset).